src/renderer/components/CanvasVisualiser/Events.js

```diff
@@ -3,89 +3,112 @@ let selectedNodes = null;
 
 export function dimNotHighlightedNodesAndEdges(connectedNodes, connectedEdges) {
   const nodesToUpdate =
-  this.getNode()
-    .filter(x => !connectedNodes.includes(x.id))
-    .map(x => ({ id: x.id, color: x.color.dimmedColor, font: { color: x.font.dimmedColor } }))
-    .reduce(collect, []);
+    this.getNode()
+      .filter(x => !connectedNodes.includes(x.id))
+      .map(x => ({ id: x.id, color: x.color.dimmedColor, font: { color: x.font.dimmedColor } }))
+      .reduce(collect, []);
 
   const edgesToUpdate =
-  this.getEdge()
-    .filter(edge => !connectedEdges.includes(edge.id))
-    .map(edge => ({ id: edge.id,
-      color: {
-        color: edge.color.color.replace('1)', '0.2)'),
-        highlight: edge.color.highlight,
-        hover: edge.color.hover,
-      } }))
-    .reduce(collect, []);
-
+    this.getEdges()
+      .filter(edge => !connectedEdges.includes(edge.id))
+      .map(edge => ({
+        id: edge.id,
+        color: {
+          color: edge.color.color.replace('1)', '0.2)'),
+          highlight: edge.color.highlight,
+          hover: edge.color.hover,
+        },
+      }))
+      .reduce(collect, []);
   this.updateEdge(edgesToUpdate);
   this.updateNode(nodesToUpdate);
 }
 
 export function setDefaultColoursOnDimmedElements(connectedNodes, connectedEdges) {
   const nodesToUpdate =
-  this.getNode()
-    .filter(x => !connectedNodes.includes(x.id))
-    .map(x => ({ id: x.id, color: x.colorClone, font: x.fontClone }))
-    .reduce(collect, []);
+    this.getNode()
+      .filter(x => !connectedNodes.includes(x.id))
+      .map(x => ({ id: x.id, color: x.colorClone, font: x.fontClone }))
+      .reduce(collect, []);
 
   const edgesToUpdate =
-  this.getEdge()
-    .filter(x => !connectedEdges.includes(x.id))
-    .map(edge => ({ id: edge.id,
-      color: {
-        color: edge.color.color.replace('0.2)', '1)'),
-        highlight: edge.color.highlight,
-        hover: edge.color.hover,
-      } })).reduce(collect, []);
+    this.getEdges()
+      .filter(x => !connectedEdges.includes(x.id))
+      .map(edge => ({
+        id: edge.id,
+        color: {
+          color: edge.color.color.replace('0.2)', '1)'),
+          highlight: edge.color.highlight,
+          hover: edge.color.hover,
+        },
+      })).reduce(collect, []);
```

Comment:
I know this was already here, but for future reference: we can remove all the `.reduce(collect, [])`, it's redundant because `.map` already returns an array

Reply:
cool, will remove them